cli/cmd/proxy/stats/command.go

```diff
@@ -50,22 +63,29 @@ func (c *StatsCommand) init() {
 
 	f := c.set.NewSet("Command Options")
 	f.StringVar(&flag.StringVar{
-		Name:    "namespace",
+		Name:    flagNameNamespace,
 		Target:  &c.flagNamespace,
 		Usage:   "The namespace where the target Pod can be found.",
 		Aliases: []string{"n"},
 	})
+	f.StringVar(&flag.StringVar{
```

Comment:
My understanding so far is that this can just be a boolean instead of a string only accepting `archive` as an input. A better flag can be `--output <true|false>`. Is there any reason why it's a specific string `archive` ? Are there plan in the future for extending this?

Even if we have plans of extending this, outputFormat should be an actual format like `json` or `hcl`. I'm not clear about the usage of `archive` here.